packages/components/src/components/Table/Table.js

```diff
@@ -109,26 +109,27 @@ function getTranslateWithId(intl) {
   };
 }
 
-const Table = props => {
+const emptyArray = [];
+
+const Table = ({
+  batchActionButtons = emptyArray,
+  className,
+  emptyTextAllNamespaces = null,
+  emptyTextSelectedNamespace = null,
+  filters,
+  hasDetails,
+  headers: dataHeaders,
+  id = null,
+  isSortable = false,
+  loading = false,
+  rows: dataRows,
+  selectedNamespace = null,
+  size = hasDetails ? 'xl' : undefined,
+  skeletonRowCount,
+  title = null,
+  toolbarButtons = emptyArray
```

Comment:
Same question but for Arrays, any chance that setting `batchActionButtons` and `toolbarButtons` to the same initial array could result in cross-contamination when values are added to either array?

Reply:
See comment above